src/create-schedule/components/Categories.tsx

```diff
@@ -0,0 +1,33 @@
+import { CATEGORY_TAGS } from "@create-schedule/constants";
+import { CategoryTags } from "@shared/types";
+import CategoryTagBox from "./CategoryTagBox";
+import ScheduleTitle from "./ScheduleTitle";
+
+interface CategoriesProps {
+  clickedCategory: CategoryTags;
+  handleClickCategory: (value: CategoryTags) => void;
+}
+
+const Categories = ({
+  clickedCategory,
+  handleClickCategory,
+}: CategoriesProps) => {
+  return (
+    <>
+      <ScheduleTitle title="카테고리" />
+      <div className="flex flex-wrap w-[414px] gap-[6px]">
```

Comment:
```suggestion
      <div className="flex flex-wrap w-[414px] gap-1.5">
```

가급적이면 스태틱한 입력보단 em으로 대체가 가능하면 이 방식으로 가는게 맞을것같습니다~

Reply:
넵 해당 부분 전체 수정하도록 하겠습니다!